spec.html

```diff
@@ -32137,11 +32237,47 @@ <h1>
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It converts _t_ from local time to a UTC time value.</dd>
+          <dd>
+            It converts _t_ from local time to a UTC time value.
+            The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.
+          </dd>
         </dl>
         <emu-alg>
-          1. Return _t_ - LocalTZA(_t_, *false*).
-        </emu-alg>
+          1. Let _localTimeZone_ be DefaultTimeZone().
+          1. Let _offsetParseResult_ be Completion(ParseTimeZoneOffsetString(_localTimeZone_)).
+          1. If _offsetParseResult_ is a normal completion, then
+            1. Let _offsetNs_ be _offsetParseResult_.[[Value]].
+          1. Else,
+            1. Let _possibleInstants_ be GetNamedTimeZoneEpochNanoseconds(_localTimeZone_, ℝ(YearFromTime(_t_)), ℝ(MonthFromTime(_t_)) + 1, ℝ(DateFromTime(_t_)), ℝ(HourFromTime(_t_)), ℝ(MinFromTime(_t_)), ℝ(SecFromTime(_t_)), ℝ(msFromTime(_t_)), 0, 0).
+            1. If _possibleInstants_ is not empty, then
+              1. Let _disambiguatedInstant_ be _possibleInstants_[0].
+            1. Else,
+              1. NOTE: _t_ represents a local time skipped at a positive time zone transition (e.g. due to daylight saving time starting or a time zone rule change increasing the UTC offset).
+              1. Let _possibleInstants_ be GetNamedTimeZoneEpochNanoseconds(_localTimeZone_, ℝ(YearFromTime(_tBefore_)), ℝ(MonthFromTime(_tBefore_)) + 1, ℝ(DateFromTime(_tBefore_)), ℝ(HourFromTime(_tBefore_)), ℝ(MinFromTime(_tBefore_)), ℝ(SecFromTime(_tBefore_)), ℝ(msFromTime(_tBefore_)), 0, 0), where _tBefore_ is the largest integral Number &lt; _t_ for which _possibleInstants_ is not empty (i.e., _tBefore_ represents the last local time before the transition).
+              1. Let _disambiguatedInstant_ be the last element of _possibleInstants_.
```

Comment:
What's the motivation for choosing the last element here but the first element earlier?